### ISSUE TITLE
since opnsense 23.7.7 updae os-unifi7-maxit install failed because of missing dependency

### ISSUE THREAD

**Issue by ro-r** [mimugmail/opn-repo#184]
pkg install os-unifi7-maxit
Updating OPNsense repository catalogue...
OPNsense repository is up to date.
Updating mimugmail repository catalogue...
mimugmail repository is up to date.
All repositories are up to date.
**pkg: libevent has a missing dependency: openssl
pkg: mongodb36 has a missing dependency: openssl**

https://github.com/opnsense/changelog/commit/69f363014857399ddaf312147b195a6ff7f88f63

"The first change is the move of the "openssl" package to "openssl111" since the former is now based on version 3.
This can and likely will disrupt updates of third party packages not having
followed this change."

**Comment by mimugmail** [comment]
Yep, I'm currently building against 111, but it will break every existing unifi install .. :/

**Comment by mimugmail** [comment]
For reference
https://forum.opnsense.org/index.php?topic=36641.0

New build was successful, but I need a bit to compare all packages

**Comment by wi23avb** [comment]
After upgrade to 23.7.7 and to latest unifi package, the unifi controller throws 404 error. If I restart the service it constantly crashes until I reboot.

**Comment by ro-r** [comment]
unfortunately after update to OPNsense 24.1 dependencys missing again
could you please build against new openssl ?

pkg install os-unifi7-maxit
Updating OPNsense repository catalogue...
OPNsense repository is up to date.
Updating mimugmail repository catalogue...
mimugmail repository is up to date.
All repositories are up to date.
pkg: libevent has a missing dependency: openssl111
pkg: mongodb44 has a missing dependency: openssl111
